### PR TITLE
IE8

### PR DESCRIPTION
**Pull request by Rich-Harris**
**not ready for merge**

IE8 support needs a bit of work... there's a lot of red in the test suite, though a lot of the failures are typical IE8 nonsense like weird treatment of whitespace, as opposed to actual problems.

### CODE REVIEW

**Comment by evs-chris** [comment]
You, sir, are a saint.

**Comment by Rich-Harris** [comment]
I'm not thinking very saintly thoughts right now

**Comment by martypdx** [comment]
image

**Comment by Rich-Harris** [comment]
Right, I'm done. Of the 2631 tests in the test suite, 2109 are capable of running in IE8 (i.e. they don't depend on ES5 accessors, or SVG, or something else unsupported). Of those, 2078 pass and 31 fail. All the failures are harmless:

* tests for unsupported CSS (nth-child, etc)
* borked whitespace (an IE8 special skill)
* programmatically-fired events not having the desired secondary effects (i.e. they're impossible to test in an automated way)
* attributes that IE8 doesn't support (e.g. `spellcheck`)

In other words, Ractive 0.7.0 supports IE8

**Comment by evs-chris** [comment]
Now there's one more miracle attributed to you. You've fastened the helmet, screwed the lid back on the paste, and won the internet for today!

**Comment by Madgvox** [comment]
Wow, nice job! :clap:
